Source/Flow/Public/FlowTypes.h

```diff
@@ -25,7 +25,10 @@ enum class EFlowNodeState : uint8
 	NeverActivated,
 	Active,
 	Completed,
-	Aborted
+	Aborted,
+
+	Max UMETA(Hidden),
+	Invalid UMETA(Hidden),
```

Comment:
I needed an invalid value for some testing I was doing using EFlowNodeState as my example test enum type.